### DISCUSSION TITLE
Friends not highlighted in global and gameplay leaderboards

### DISCUSSION THREAD

**Discussion by smileyhead** [ppy/osu#24724] [Ideas]
Stable highlights your friends on the global leaderboard and during gameplay on the left-side leaderboard. Having this feature added back would be nice.

**Comment by cdwcgt** [comment]
![image](https://github.com/ppy/osu/assets/34775378/8a9ec675-0a95-4bf4-80ed-7d963f7d66be)
something like this?

**Reply by peppy** [reply]
Maybe you need a better screenshot. I'm guessing you just made everyone a friend? But this doesn't give a good idea of how this will look in actual usage.

**Reply by cdwcgt** [reply]
no, this is my friend leaderboard

here is global leaderboard
![image](https://github.com/ppy/osu/assets/34775378/4009a5be-17e3-48d6-9024-1615ae9559ab)

~~find a friend on global leaderboard is difficult~~

**Reply by peppy** [reply]
So make a test scene? If you're planning on implementing the feature you will need it anyway.

**Reply by cdwcgt** [reply]
sure

![image](https://github.com/ppy/osu/assets/34775378/1b2d91c1-18d8-42de-a692-899bcade6097)
